cmd/podman/build.go

```diff
@@ -187,6 +187,13 @@ func buildCmd(c *cli.Context) error {
 	}
 	namespaceOptions.AddOrReplace(usernsOption...)
 
+	layers := buildahcli.UseLayers()
```

Comment:
Do we need this? We're unconditionally overwriting it with either c.Bool("layers") or true

Reply:
@mheon I think we do. I needed an intuitive set of cases for the end user. I tested these cases:
1. podman --layers=true (overrides everything)
2. podman --layers=false (overrides everything)
3. export BUILDAH_LAYERS=true (works, but is overridden by podman --layers=false)
4. export BUILDAH_LAYERS=false (works, but is overridden by podman --layers=true)

c.Bool("layers") gets us the command line option, but then sets the default for podman differently with an else statement. That's why the is.Set logic is there....

Reply:
Also, 5. defaults to true...